app/v2/notifications/get_notifications.py

```diff
@@ -55,7 +55,7 @@ def get_pdf_for_notification(notification_id):
     except Exception:
         raise PDFNotReadyError()
 
-    return send_file(filename_or_fp=BytesIO(pdf_data), mimetype='application/pdf')
+    return send_file(path_or_file=BytesIO(pdf_data), mimetype='application/pdf')
```

Comment:
why has this argument name changed? Also something to do with dependency updates?

Reply:
The send_file method has changed in the latest version of Flask